scrambler.cpp

```diff
@@ -647,10 +654,15 @@ void print_node(std::ostream &out, const scrambler::node *n, annotation_mode kee
             if (gen_ucore) {
                 // insert (get-unsat-core) after each check-sat
                 out << std::endl << "(get-unsat-core)";
-            } else if (gen_mval) {
+            }
```

Comment:
I removed the else here.
I guess if someone specifies multiple of these options, he really wants to have all corresponding get-unsat-core/get-mode/get-proof commands.